pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.6.0,<4"
-datadog = "^0.41.0"
+datadog = "^0.41"
 wrapt = "^1.11.2"
 ddtrace = "^0.61.1"
```

Comment:
If this change is considered acceptable, we may also want to make a similar change for `ddtrace` which is also effectively locked to `0.61.y` (y >= 1). I'd be happy to make that change if requested.